src/components/Header/Header.js

```diff
@@ -73,23 +78,18 @@ export default class Header extends React.Component {
     }
     this.setState({ isProfileOpen: false, isStatusOpen: false, isGuideOpen: false })
   }
-
   clickBook(e) {
     this.setState({ isGuideOpen: !this.state.isGuideOpen, isStatusOpen: false, isProfileOpen: false })
   }
-
   handleStatusEnter() {
     this.setState({ isStatusHover: true })
   }
-
   handleStatusLeave() {
     this.setState({ isStatusHover: false })
   }
-
-
   render() {
     return (
-      <>
+      <div>
```

Comment:
The root node has to be a valid html element, not a virtual node, in order for this to work